client/lib/notifications/note-block-parser.js

```diff
@@ -117,6 +117,13 @@ const userNode = ( { id: userId, name, site_id: siteId } ) => ( {
 	userId,
 } );
 
+const pluginNode = ( { site_slug, slug, version } ) => ( {
+	type: 'plugin',
+	siteId: site_slug,
```

Comment:
noted in D10275, but it's a good idea to clarify what `site_slug` here is. it's _not_ the `siteId` regardless so we shouldn't conflate those terms.

passing the `siteId` in the API response or passing both the `siteId` and the `slug`/`domain` is another reasonable way to resolve ambiguity. mainly we can ask, do we want to preserve the slug-representation of the site in the URL or is a site id good enough…

Reply:
Right, the `client/components/notes-formatted-block/index.jsx` currently uses:
```
return <a href={ `/plugins/${ name }/${ siteId }` }>{ descent }</a>;
```
so I think I should update it and use `siteSlug` everywhere.